### DISCUSSION TITLE
推薦好用的擋廣告app

### DISCUSSION THREAD

**Discussion by pekoranoinu** [1101-GS4719-C/Python-Community-GS4719-C#395] [General]
我知道大家都在用youtube，但有時廣告跳出來會影響影片觀賞的品質，所以我推薦大家使用這個擋廣告app，讓大家看youtube時有更棒的觀賞品質。(非業配)
https://chrome.google.com/webstore/detail/adblock-for-youtube/cmedhionkhpnakcndndgjdbohmhepckk?hl=zh-TW
![圖片](https://user-images.githubusercontent.com/99986220/156401841-0e7fdf1c-a1a9-46e2-b389-89c225788596.png)

**Comment by lx3211** [comment]
如果用MacOS 的話 adguard 也很好用喔！

**Comment by chupochun** [comment]
原來youtube的廣告是可以擋住的 這樣以後看影片就不需要等廣告了 謝謝分享

**Comment by JunTingLin** [comment]
那這邊來推薦一個手機youtube去廣告的方法😁
[🎫不用付錢!就可以享受YouTube Premium #399](https://github.com/NCU-GS4719-Python/Python-Community/discussions/399)

**Comment by JSHT** [comment]
可以使用 AdblockPlus，不只可以擋住 youtube 廣告，也可以擋住各網頁的惱人廣告
![image](https://user-images.githubusercontent.com/48956859/156709323-ade75912-0cb9-4267-b34a-84687e1bad60.png)

**Reply by chia-kai-chang** [reply]
我用這一款plug-in 哈~

**Reply by bloodtrail0511** [reply]
我本身也在用這個!他們最近還有adblock vpn，更容易保護自己的隱私

**Comment by yuchi06274** [comment]
感謝分享 真的非常實用

**Comment by Rosancl** [comment]
這個好實用! 馬上安裝~~ 謝謝分享 👍

**Comment by xkllkx** [comment]
我是windows用戶，我也是用adguard，真的很好用！youtube vanced也很實用

**Comment by yucheng1028** [comment]
身為願意支持影片創作者的我，每個月都少喝幾杯飲料買youtube premium。因為使用ad blocker 影片創作者可能會缺少廣告收益 : (
但還是感謝分享 ~

**Comment by Ku-Hao-Chen** [comment]
感謝分享，往後可以提升影片觀賞的品質。

**Comment by awkward-willy** [comment]
自己也有使用一樣的 ad blocker
透過建立自己的播放清單，搭配使用
就能達到無廣告且自訂撥放的歌單
是學生黨的好選擇
（當然可能出現創作者營利減少的問題...）

**Comment by jason79461385** [comment]
後來都關掉了，幫喜歡的創作者看看廣告也不錯

**Comment by stan890425** [comment]
我記得之前的Ad blocker可以擋其他網站的廣告

**Comment by yi-1111** [comment]
Ad blocker可以擋其他網站的廣告，還可以手動處理特定的廣告，蠻好用的

**Comment by Jimmy911218** [comment]
這個確實可以黨管告，但有時候會直接管告聲音螢幕黑平不讓人跳過

**Comment by Ben6uo** [comment]
確實 平時看到15秒無法skip的廣告是真的氣死

**Comment by leon10456** [comment]
不管什麼領域AI運算都很重要呢

**Comment by 110409538** [comment]
那以後用youtube聽歌就不會被廣告干擾了.

**Comment by HsuChen88** [comment]
使用這個外掛套件也要注意隱私的問題..
我本來也有用 但現在就不太敢用了

**Comment by Lai27330** [comment]
感謝分享，每次看到那個一直在亞馬遜上買多少多少的廣告就很煩躁，終於可以擋住了。

**Comment by HaoFish123** [comment]
一直在找有哪個好用的，感謝分享。

**Comment by treeann** [comment]
一直在追尋阿終於有人推薦了˙

**Comment by yucheng1028** [comment]
這會不會很吃電腦效能阿

**Comment by sys511613** [comment]
擴充元件真的很好用，但是要注意有時候可能會有一些病毒
之前裝了一個套件，結果過幾天發現youtube多了一堆奇怪的觀看紀錄，原本還以為是帳號被盜，改密碼後發現觀看紀錄還是一直更新，才發現是插件的問題

**Comment by jcpan821** [comment]
現在的廣告很多都強制兩部，這個真的省很多時間

**Comment by alexthebigpp** [comment]
實用好物，感謝分享。

**Comment by yenwei12138** [comment]
之前就因為太煩廣告買了premium，現在才知道竟然還有這種東西......

**Comment by etin2468** [comment]
adblock真的讚，雖然手機等行動裝置用不了

**Comment by Lynn12378** [comment]
感謝樓主分享!超實用的
